### ISSUE TITLE
Incorrect Panic Warning for Option.unwrap()

### ISSUE THREAD

**Issue by stoneman258** [lizhuohua/rust-mir-checker#18]
## Description
When using MirChecker to detect bugs in my project, it reports many false panic warnings. The minimized example is listed below. The`is_some` API is a very popular in Rust to avoid null dereference, and no panic can be triggered in this program.

### Minimized Code Example
```rust
fn extract(val: Option<i32>) -> i32 {
    if val.is_some() {
        val.unwrap() //should not report warning
    } else {
        0
    }
}
```

## Expected Behavior
No false warning is reported.

## Actual Behavior
MirChecker reports a panic warning at `rustlib/src/rust/library/core/src/option.rs:385:18`.

**Comment by stoneman258** [comment]
@lizhuohua Could you please help me check and solve the issue?
